### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by stefantsonev2019**
I think we need a README file.

### CODE REVIEW

**Comment by stefantsonev2019** [comment]
As an addition to the changes in #1

**Comment by krasimir** [comment]
Yep. I agree we should add a README file.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# GitHorn playground
+
+A repo for testing while developing GitHorn
```
Comment (krasimir) [inline README.md]: Shall we add more description here?

**Comment by krasimir** [comment]
Overall it looks good.

File: README.md (hunk 1 above)
Comment (stefantsonev2019) [reply README.md]: Yep. I'll do.

Comment (krasimir) [reply README.md]: Ok, thanks.

**Comment by krasimir** [comment]
That's a new comment here.

**Comment by krasimir** [comment]
AAA

**Comment by krasimir** [comment]
A new one

**Comment by krasimir** [comment]
OKL

**Comment by krasimir** [comment]
GHH

**Comment by krasimir** [comment]
hjsa22

**Comment by krasimir** [comment]
33

**Comment by krasimir** [comment]
oi

**Comment by krasimir** [comment]
psps

**Comment by krasimir** [comment]
gtr

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# GitHorn playground
```
Comment (krasimir) [inline README.md]: A thread

Comment (krasimir) [reply README.md]: With a comment.

Comment (stefantsonev2019) [reply README.md]: A-ha

Comment (krasimir) [inline README.md]: Again

Comment (stefantsonev2019) [reply README.md]: ddd

Comment (krasimir) [reply README.md]: u7u7

Comment (krasimir) [inline README.md]: 1

Comment (stefantsonev2019) [reply README.md]: 2

File: README.md (hunk 1 above)
Comment (krasimir) [reply README.md]: something to delete

File: README.md (hunk 2 above)
Comment (krasimir) [inline README.md]: KKK

Comment (stefantsonev2019) [reply README.md]: 1

Comment (krasimir) [reply README.md]: ddd

Comment (krasimir) [inline README.md]: JJJ

Comment (stefantsonev2019) [reply README.md]: ggg

Comment (krasimir) [reply README.md]: ttt

Comment (krasimir) [reply README.md]: 1

Comment (krasimir) [reply README.md]: 1

Comment (krasimir) [reply README.md]: 2

Comment (krasimir) [reply README.md]: 3

Comment (krasimir) [inline README.md]: now

Comment (krasimir) [reply README.md]: 2223

Comment (krasimir) [reply README.md]: new comment

Comment (krasimir) [reply README.md]: kki

Comment (krasimir) [reply README.md]: Hello

Comment (krasimir) [reply README.md]: New one

Comment (krasimir) [reply README.md]: 6555

Comment (krasimir) [reply README.md]: 888

Comment (krasimir) [reply README.md]: Mon

Comment (krasimir) [reply README.md]: Thu

File: README.md (hunk 1 above)
Comment (krasimir) [reply README.md]: To review

[the remark this answers is not in the document]
Comment (krasimir) [reply README.md]: And it works yes.

**Comment by krasimir** [comment]
A single comment.

**Review by krasimir — APPROVED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (krasimir) [inline README.md]: A new review has been started.

Comment (krasimir) [inline README.md]: A single comment.

[the remark this answers is not in the document]
Comment (krasimir) [reply README.md]: One more!

Comment (stefantsonev2019) [inline README.md:3]: Do  you think that we ...

Comment (krasimir) [reply README.md]: Yes.

Comment (krasimir) [reply README.md]: And one more here.

Comment (krasimir) [reply README.md]: YYY

Comment (krasimir) [reply README.md]: TTT

Comment (krasimir) [reply README.md]: I think so.

Comment (krasimir) [reply README.md]: and 8888

File: README.md (hunk 2 above)
Comment (krasimir) [inline README.md]: Ok let's see

Comment (krasimir) [inline README.md]: Hell

Comment (krasimir) [reply README.md]: ggh

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (krasimir) [inline README.md]: OK

Comment (krasimir) [inline README.md]: Empty

Comment (krasimir) [inline README.md]: YYYYY

Comment (krasimir) [inline README.md]: Yep

Comment (krasimir) [inline README.md]: Now this should work.

Comment (krasimir) [inline README.md:2]: Oh nice

File: README.md (hunk 1 above)
Comment (stefantsonev2019) [reply README.md:3]: uuu

File: README.md (hunk 3 above)
Comment (stefantsonev2019) [reply README.md:2]: yuyu

**Review by krasimir — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (krasimir) [reply README.md]: ddd

**Review by krasimir — CHANGES_REQUESTED** [review]
no no asdsda
```
Hello
```

File: README.md (hunk 1 above)
Comment (krasimir) [reply README.md:3]: a new one

Comment (krasimir) [reply README.md:3]: joker

**Review by krasimir — APPROVED** [review]
Hm ... yes it may work indeed

File: README.md (hunk 1 above)
Comment (krasimir) [reply README.md:3]: one m

**Review by krasimir — CHANGES_REQUESTED** [review]
I think that it is fine

File: README.md (hunk 3 above)
Comment (krasimir) [reply README.md]: 9090909090

Comment (krasimir) [reply README.md:2]: dsa

**Review by krasimir — COMMENTED** [review]
nono34

**Review by krasimir — APPROVED** [review]
Good work!

**Review by krasimir — CHANGES_REQUESTED** [review]
nope11

**Review by krasimir — COMMENTED** [review]
I have a few comments.

**Pull request closed without merging**